scripts/ldap/schema2ldif.pl

```diff
@@ -0,0 +1,44 @@
+#!/usr/bin/env perl
```

Comment:
Please add a copyright header to this file.

Reply:
As in `Copyright (c) 2018 Red Hat, Inc.`?

Reply:
I'm not really an expert either, but the primary reason is to have each contributed file licensed properly and a reference to it's author(s). Another reason is that for example the Debian Packaging Guidelines are very strict about the licensing stuff and require (AFAIK) that every project file has a license header.

So if you are the author of the script, please record that in the header. Choose whatever OSI compatible license you wish. The easiest route is to go with what USBGuard uses itself.

Reply:
Since the original is already available with a MIT license header in other locations, after some research I decided to keep that here as well.